signalfx-org-metrics/docs/sf.org.limit.dataPointsPerMinute.md

```diff
@@ -0,0 +1,10 @@
+---
+title: sf.org.limit.dataPointsPerMinute
+brief: Upper limit of the DPM in the organization
+metric_type: gauge
+---
+### sf.org.limit.dataPointsPerMinute
+
+Upper limit of how many data points per minute that can be sent to SignalFx in the organization.
```

Comment:
Is this also accurate? If so, it might be clearer.

Maximum number of datapoints per minute (DPM) SignalFx will process and store.

(FYI, this is the language in the related FAQ: https://docs.signalfx.com/en/latest/_sidebars-and-includes/dpm-faq.html  When pcl comes back from vacation, I will ask him if he thinks we should add a link to that doc.)

note: no need to say "in the organization" as these are org metrics by definition. OK to remove throughout.